data/transactions/logic/langspec.json

```diff
@@ -15,8 +18,12 @@
     {
       "Opcode": 1,
       "Name": "sha256",
-      "Args": "B",
-      "Returns": "B",
+      "Args": [
+        "[]byte"
+      ],
+      "Returns": [
+        "hash"
+      ],
```

Comment:
@algorandskiy is there some plugin that understands these "BBB" and "UU" type things?